src/theme/base/_mediaqueries.scss

```diff
@@ -69,6 +71,11 @@ $break-large: 992px;        // tablets
         @content;
     }
   }
+  @else if $media==largepc {
```

Comment:
With #123 merged, we no longer need this, can you please update your code accordingly? Thanks!

Reply:
That's why I said in frontend channel that until we got the new mediaqueries.. everyone should have used manual mediaqueries doing mobile-first. Now is going to be a pain in the ass for some people :disappointed: 

For any help, reach me on slack (victor ribero)

Reply:
😞